unity-client/Assets/Scripts/MainScripts/DCL/Components/LoadableShapes/NFTShape/NFTShapeLoaderController.cs

```diff
@@ -87,19 +87,17 @@ void Awake()
         InitializePerlinNoise();
     }
 
-    void Update()
-    {
-        hqTextureHandler?.Update();
-    }
+    void Update() { hqTextureHandler?.Update(); }
```

Comment:
I have seen this "one line function" style in several PRs... is this being applied by the new automated code-style pre-commits? I guess it is something very personal, but I'm not a big fan of this kind of style, even if the function has only 1 line.

Reply:
Yeah, is the automated code-style. I'm not a big fan of it either but after several tries to fix it without messing anything else up, I gave up hahaha. 

I can give it another try in the near-future.